base/rawbigints.jl

```diff
@@ -26,8 +25,9 @@ split_bit_index(x::RawBigInt, i::Int) = divrem(i, word_length(x), RoundToZero)
 `i` is the zero-based index of the wanted word in `x`, starting from
 the less significant words.
 """
-function get_elem(x::RawBigInt, i::Int, ::Val{:words}, ::Val{:ascending})
-    unsafe_load(x.d, i + 1)
+function get_elem(x::RawBigInt{T}, i::Int, ::Val{:words}, ::Val{:ascending}) where {T}
+    # `i` must be nonnegative and less than `x.word_count`
+    unsafe_load(Ptr{T}(pointer(x.d)), i + 1)::T
```

Comment:
Need a `GC.@preserve` on `x.d` here?

Reply:
Ah, OK, thank you. I guess I assumed it's not necessary in such a short one-liner.